componentTemplates/linters/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We need a story to remove these from the Generator and then the Generator will require a dependency on this package with a script to install its contents.